orangecontrib/bio/genialis/genapi.py

```diff
@@ -0,0 +1,190 @@
+from genesis import Genesis
+from genesis import GenData
+import json
+from Orange.data import *
+from .. import dicty
+import io
+import gzip
+
+DEFAULT_EMAIL = 'anonymous@genialis.com'
+DEFAULT_PASSWD = 'anonymous'
+DEFAULT_URL = 'https://dictyexpress.research.bcm.edu'
+CallBack = dicty.CallBack
+
+
+class Genapi(object):
```

Comment:
I would rename this to something resolwe-like, say ResolweServer. The goal of this class is to connect to both platforms (Genesis and Resolwe), right?